src/paymentservice/index.js

```diff
@@ -1,59 +1,68 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
-const grpc = require('@grpc/grpc-js')
```

Comment:
@Kimbohlovette could you keep the same indentation we had before?

It would make our life as reviewers easier

Reply:
okay

Reply:
okay